tests/unit/package_managers/test_npm.py

```diff
@@ -584,10 +584,12 @@ def test_get_purl_for_local_package(
         rooted_tmp_path: RootedPath,
         mock_get_repo_id: mock.Mock,
     ) -> None:
+        from hermeto.core.constants import Mode
```

Comment:
This should be a module-level import unless there is a very good reason for it not to be.